django/db/migrations/state.py

```diff
@@ -346,6 +346,36 @@ def _reload(self, related_models):
         # Render all models
         self.apps.render_multiple(states_to_be_rendered)
 
+    def resolve_model_field_relations(
+        self, model_key, field_name, field, concretes=None,
+    ):
+        remote_field = field.remote_field
+        if not remote_field:
+            return
+        if concretes is None:
```

Comment:
Do you need to handle `concretes` being `None` in this PR since your only caller below is already doing those lines if `concretes` is `None`?

Reply:
We don't need it here but it will he used heavily in #14587. It should be fine to do this in advance.